roles/confluent.control_center/tasks/main.yml

```diff
@@ -61,6 +61,15 @@
     group: "{{control_center_group}}"
     recurse: true
 
+# Archive File deployments need to create SystemD directory on Debian
+- name: Create Zookeeper Service path
```

Comment:
I would suggest to put this repetitive task in common-role. Don't we already have OS-specific tasks there?

Reply:
Really good idea!  Will change shortly.